src/registry/routes/helpers/get-component.js

```diff
@@ -149,7 +150,12 @@ module.exports = function(conf, repository){
           renderMode: renderMode
         });
 
-        if(isUnrendered){
+        if (skipRendering) {
+          callback({
+            status: 200,
+            response: response
```

Comment:
This includes `renderMode` which is not actually very relevant here. Would you mind omitting that property in case `skipRendering===true`?